docs/framework/wcf/samples/wcf-analytic-tracing.md

```diff
@@ -47,7 +47,7 @@ This sample demonstrates how to add your own tracing events into the stream of a
   
 4.  Run the WCF test client (WcfTestClient.exe).  
   
-     The WCF test client (WcfTestClient.exe) is located in the \<Visual Studio 2012 Install Dir>\Common7\IDE\ WcfTestClient.exe (default Visual Studio 2012 install dir is C:\Program Files\Microsoft Visual Studio 10.0).  
+     The WCF test client (WcfTestClient.exe) is located at `\<Visual Studio 2012 Install Dir>\Common7\IDE\WcfTestClient.exe` (default Visual Studio 2012 install dir is `C:\Program Files\Microsoft Visual Studio 10.0`).  
```

Comment:
nit unrelated to your PR: The clause in parentheses should be a separate sentence -- e.g., "(The default Visual Studio 2012 install directory is...).